x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -484,13 +484,20 @@ export const EDIT_RULE_SETTINGS = i18n.translate(
   }
 );
 
-export const EDIT_RULE_SETTINGS_TOOLTIP = i18n.translate(
-  'xpack.securitySolution.detectionEngine.rules.allRules.actions.editRuleSettingsToolTip',
+export const LACK_OF_KIBANA_PRIVILEGES = i18n.translate(
```

Comment:
Thanks for renaming this constant! For more clarity, I'd propose to name it more specific, like `LACK_OF_KIBANA_ACTIONS_FEATURE_PRIVILEGES`, as Kibana privileges is a broader term that includes many different features, including actions, security, etc.